app/src/ui/components/Switch.jsx

```diff
@@ -0,0 +1,23 @@
+import styles from "./Switch.module.css";
+import clsx from "clsx";
+
+const Switch = ({ onChange, isActiveMode }) => {
```

Comment:
you dont need `app.jsx` to know about the actual state of Switch, so lets move the stat ehere

lets use 2 props here and only it:
onChange, defaultValue

just change theme on onChange - it has all of the required info inside: `event.target.value` or something like that. May be it can be `event.target.checked` i dont know